Doc/install/INSTALL.rst

```diff
@@ -111,7 +111,7 @@ UNIX/Linux
 For installing Biopython, you will require a software package called ``pip`` to do so. If you are using the latest 3.x version of Python, you will have the package preinstalled in your UNIX/Linux system. You can check the version of the software by doing::
 
   $ pip --version
-  pip 9.0.1 from /usr/lib/python3.6/site-packages (python 3.6)
+  pip 9.0.1 from /usr/lib/python3.6/site-packages (python 3.5)
```

Comment:
This looks wrong - there is a mix of Python 3.6 and 3.5 here now (the old version was all Python 3.6)